Nocilla/Matchers/LSMatcher.m

```diff
@@ -10,4 +10,15 @@ - (BOOL)matchesData:(NSData *)data {
     return [self matches:[[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding]];
 }
 
+
+#pragma mark - Equality
+
+- (BOOL)isEqual:(id)object {
+    @throw [NSException exceptionWithName:NSInternalInconsistencyException reason:@"[LSMatcher isEqual:] is an abstract method" userInfo:nil];
+}
+
+- (NSUInteger)hash {
+    @throw [NSException exceptionWithName:NSInternalInconsistencyException reason:@"[LSMatcher hash] an abstract method" userInfo:nil];
+}
+
```

Comment:
I realize that `LSMatcher` was originally implemented this way, but shouldn't this be a protocol with required methods, as opposed to an abstract base class? @luisobo